README.md

```diff
@@ -90,7 +90,7 @@ Validator                               | Description
 --------------------------------------- | --------------------------------------
 **contains(str, seed [, options ])**    | check if the string contains the seed.<br/><br/>`options` is an object that defaults to `{ ignoreCase: false, minOccurrences: 1 }`.<br />Options: <br/> `ignoreCase`: Ignore case when doing comparison, default false<br/>`minOccurences`: Minimum number of occurrences for the seed in the string. Defaults to 1.
 **equals(str, comparison)**             | check if the string matches the comparison.
-**isAfter(str [, date])**               | check if the string is a date that's after the specified date (defaults to now).
+**isAfter(str [, options])**            | check if the string is a date that's after the specified date.<br/><br/>`options` is an object that defaults to `{ comparisonDate: Date().toString() }`.<br/>**Options:**<br/>`comparisonDate`: Date to compare to. Defaults to `Date().toString()` (now).
```

Comment:
This will be a breaking change, right?

Reply:
Nope, it's only a change to the docs
They only document the new options now, but the current method still works. See also the tests that are copied. I chose to only document the new options since the README is already long enough, in #2154 we can discuss future methods of making sure all possible ways of using validators are documented.

Reply:
Got it, thanks!